src/controllers/cockpit/process.js

```diff
@@ -0,0 +1,306 @@
+const cockpitService = require("../../services/cockpit");
+const { getEngine, getCockpit } = require("../../engine");
+const { logger } = require("../../utils/logger");
+
+const setProcessState = async (ctx, next) => {
+  logger.verbose("Called setProcessState");
+
+  const cockpit = getCockpit();
+
+  const process_id = ctx.params.id;
+  const state_data = ctx.request.body;
+  const process = await cockpit.fetchProcess(process_id);
+
+  if (process) {
+    const next_node = process._blueprint_spec.nodes.find((n) => n.id === state_data.next_node_id);
+    if (next_node) {
+      const result = await setState(process, state_data);
+      ctx.status = 200;
+      //TODO: add a dryrun to see the next node execution
```

Comment:
Este to do irá ficar para a próxima tarefa ou deveria ter sido feito neste PR?

Reply:
todos os "TO DOs" são oportunidade de melhoria e não pendências.